File: shoebill_plugins/LtrpApi/src/main/java/lt/ltrp/object/InventoryEntity.java
package lt.ltrp.object;

import lt.ltrp.NamedEntity;

/**
 * @author Bebras
 *         2016.04.08.
 */
public interface InventoryEntity extends NamedEntity {

    void setInventory(Inventory inventory);
    Inventory getInventory();

}

File: shoebill_plugins/inventory/src/main/kotlin/lt/ltrp/inventory/object/impl/InventoryEntity.kt
package lt.ltrp.`object`;

/**
 * @author Bebras
 *         2016.04.08.
 *
 *         Null allowed only because Inventory creation requires InventoryEntity instance
 */
 interface InventoryEntity: NamedEntity {

    var inventory: Inventory?
        get
        set

}
